app/Enum/EventRejectionReason.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Enum;
+
+enum EventRejectionReason: string
+{
+    case DEFAULT = 'denied';
+    case LATE = 'too-late';
+    case DUPLICATE = 'duplicate';
+    case NOT_APPLICABLE = 'not-applicable';
+
+    public function getReason(): string {
```

Comment:
that doesn't return the reason? Maybe rename the function to `getTranslationKey` or similar?

Reply:
or remove this function. I don't see any reason to have this, as of the string is only used in one function... therefore we can reduce complexivity